spec/features/idv/doc_auth/socure_document_capture_spec.rb

```diff
@@ -91,6 +91,99 @@
         end
       end
 
+      context 'reuses valid capture app urls when appropriate', allow_browser_log: true do
+        before do
+          allow(IdentityConfig.store).to receive(:doc_auth_max_attempts).and_return(max_attempts)
+          (max_attempts - 1).times do
+            socure_docv_upload_documents(docv_transaction_token: @docv_transaction_token)
+          end
+        end
+
+        context 'successfully erases capture app url when flow is complete' do
+          before do
+            DocAuth::Mock::DocAuthMockClient.reset!
+          end
```

Comment:
```suggestion
        context 'successfully erases capture app url when flow is complete' do
```
is there a benefit to repeated docv submissions here?